firestore/objc/firestore-smoketest-objc/ViewController.m

```diff
@@ -277,6 +277,23 @@ - (void)updateDocument {
   // [END update_document]
 }
 
+- (void)updateDocumentArray {
+    // [START update_document_array]
+    FIRDocumentReference *washingtonRef =
+    [[self.db collectionWithPath:@"cities"] documentWithPath:@"DC"];
+
+    // Atomically add a new region to the "regions" array field.
+    [washingtonRef updateData:@{
+      @"regions": [FIRFieldValue fieldValueForArrayUnion:@[@"greater_virginia"]]
+    }];
+
+    // Atomically remove a new region to the "regions" array field.
+    [washingtonRef updateData:@{
+      @"regions": [FIRFieldValue fieldValueForArrayRemove:@[@"east_coast"]]
+    }];
+    // [END update_document_array]
+}
```

Comment:
Because of the way we initially set up the projects our ObjC snippets are two-space indents but our Swift snippets are four-space indents. Can you change these to conform?